utils/python/tests/scripts_regression_tests

```diff
@@ -130,7 +130,33 @@ def setup_proxy():
     return False
 
 ###############################################################################
-class TestWaitForTests(unittest.TestCase):
+class B_TestCreateNewcase(unittest.TestCase):
```

Comment:
because create_test supplies a number of optional arguments to create_newcase we were missing several problems that this test will pick up.